pom.xml

```diff
@@ -471,6 +476,20 @@
         <scope>test</scope>
       </dependency>
 
+      <!-- Spring REST Docs -->
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-test</artifactId>
```

Comment:
Is there any specific reason for declaring this dependency management here? It's already [provide by spring boot](https://github.com/spring-projects/spring-boot/blob/v2.0.0.M5/spring-boot-project/spring-boot-dependencies/pom.xml#L510) and it would be better to do not override the version provided by them. It's already used by the [query-service](https://github.com/Activiti/activiti-cloud-query-service/blob/develop/activiti-cloud-services-query/activiti-cloud-services-query-rest/pom.xml#L142), for instance.

Reply:
you're right, I've removed the dependencies